### ISSUE TITLE
When creating a new file and pasting, the edit window width changes

### ISSUE THREAD

**Issue by internationils** [RickStrahl/MarkdownMonster#1111]
(yay, bug #1111 ; )

When creating a new file and pasting, the edit window width changes to half size (with the right side of the main window just blank)

To reproduce:
- preview window is closed
- copy something from a .md file in the editor
- create a new file
- paste the text
- edit window resized to half-width (no preview shown on the right)
- close the tab and reopen it, everything is normal again

**Comment by RickStrahl** [comment]
I can't reproduce that.

What I see:

* Markdown doc open - preview pane closed

* ctrl-n to open new editor
* Editor opens with preview pane
* Paste text
* Text displays
* Both editor and preview pane stay exactly the same

**Comment by RickStrahl** [comment]
Ok - so I figured out the issue here which is a little more complicated and involves actually two bugs:

* The previous preview state is ignored and MM tries to refresh the preview when a new document is created  
This results in a new preview window being created which is what I saw. However, the new preview is not correctly attached to the editor, so any changes in the edit window actually fail.

* When pasting the Preview Mode is explicitly enabled  
This results in a different problem - when the preview is not already visible, it sets the preview visible flag, but it's too late in the cycle to actually render. I think this is what you see: An empty preview pane.

Two fixes were made:

* Only refresh the preview browser (and other related preview stuff) when the preview is actually visible when creating a new document

* Only disable the preview if pasting a non-markdown document. For Markdown docs, respect the active preview visibility.


That's actually a long standing issue I've been trying to pinpoint but never figured out why it occurred, so thank you for that. I think this also caused some inconsistencies in the preview browser which show up in my logs.
